src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.scss

```diff
@@ -61,3 +61,17 @@
     padding: 10px 0 20px;
   }
 }
+
+.findAGene {
+  color: $blue;
+  cursor: pointer;
+  margin: 30px 0 20px 10px;
+  width: 95px;
```

Comment:
Why do we need this, by the way? In the future, when we also add a link to open the region navigator:

![image](https://user-images.githubusercontent.com/6834224/175923979-bfbd3079-f499-4e4f-b9e2-fb37b59eb85b.png)

we would probably just use `display:flex; column-gap: something` on the parent. What's the purpose of having this width now?

Reply:
will `display: inline-block` help?

Reply:
> It was getting set to 285px by default.

Strangely it was caused by `margin: 30px 0 20px 10px;`. Changing it to `margin: 30px auto 20px 10px;` fixes it.

Reply:
> Strangely it was caused by margin: 30px 0 20px 10px;

That's because of the `display: flex` on the parent. Add `align-items: flex-start` to the parent container; and you can switch your margin-right back to 0;

Reply:
Thanks. That's done